### PR TITLE
feat: author component

### PR DESCRIPTION
**Pull request by Sepush**
Signed-off-by: Sepush <sepush@outlook.com>

### CODE REVIEW

**Comment by gaocegege** [comment]
image

Could we move the author section to the last or the right section?

**Comment by Sepush** [comment]
> image
> 
> Could we move the author section to the last or the right section?

if you want put to the last just move `<Author />` to the last of the paper

**Comment by gaocegege** [comment]
SGTM

File: docs/blog/welcome.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+---
+author: 'Gao Ce'
+avatar: 'https://www.github.com/gaocegege.png'
+github: 'https://github.com/gaocegege'
+twitter: 'https://twitter.com/gaocegege'
+introduction: 'Do cool things that matter | Co-founder @tensorchord | Co-chair @kubeflow | ex-@Tencent, @bytedance, @caicloud'
+---
+<Author/>
```
Comment (gaocegege) [inline docs/blog/welcome.md:8]: Then could you please move it to the last? Then I will merge it ASAP.

Thanks for your contribution! :tada: :+1:

**Review by gaocegege — APPROVED** [review]

**Pull request merged**